rasa/cli/arguments/data.py

```diff
@@ -74,6 +74,8 @@ def set_migrate_arguments(parser: argparse.ArgumentParser) -> None:
 
     add_out_param(
         parser,
-        default=DEFAULT_DOMAIN_PATH,
-        help_text="Path (for `yaml`) where to save migrated domain in Rasa 3.0 format.",
+        default=None,
+        help_text="Path (for `yaml`) where to save migrated domain in Rasa 3.0 format."
+        "If none is specified, a generic file/folder will be created in the folder "
```

Comment:
This can even be more specific regarding the names of the generated `out_path`:
```suggestion
        "If none is specified, either a `new_domain.yml` file or `new_domain` folder will be created in the folder "
```